workflows/update_tags.sh

```diff
@@ -102,7 +102,12 @@ function update_tags_in_file() {
   LATEST_TAG=$2
   THIS_FILE=$3
   echo "Updating tag of ${THIS_IMAGE} in ${SCRIPT_DIR}/${THIS_FILE} to ${LATEST_TAG}."
-  sed -i -e "s|${THIS_IMAGE}:.*|${THIS_IMAGE}:${LATEST_TAG}|g" $THIS_FILE
+  if [[ ${THIS_IMAGE} =~ cray-sat$ ]]; then
+    # Use a more flexible sed replacement for cray-sat images
+    sed -i -e "s|[^ ]*cray-sat[^ ]*|${THIS_IMAGE}:${LATEST_TAG}|g" "$THIS_FILE"
```

Comment:
Might want to consider whether this is _too_ general, but I don't see any other matches for this in the file that references the cray-sat image as of now, so it's probably pretty safe.

Reply:
Yeah, thats true. I did have the same thought. But since no other matches existed in the file, thought to have it this way.